### PR TITLE
29 geojson for tax lot with tax lot schema as properties and wgs84 as geometry

### LINKED ISSUE #29: Implementation of tax lot adjacent endpoints (Mikro-orm)
# Description
Implement the tax lot adjacent endpoints

## Acceptance criteria
- [x] Endpoints exist that fulfills the OpenApi description outlined in #26 
  - Omit tests to expedite development

## Related
- blocked by #26

### PR DESCRIPTION
**Pull request by horatiorosa**
## Description
- Implemented api endpoints that returns 
  - geojson for specified tax lot with tax lot schema as properties and wgs84 coordinates as geometry
  - a list of schema for zoning districts that spatially intersect the tax lot

## Tickets
- closes #29

### CODE REVIEW

File: src/app.service.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+import { EntityManager } from "@mikro-orm/postgresql";
 import { Injectable } from "@nestjs/common";
 
 @Injectable()
 export class AppService {
+  constructor(
+    private readonly orm: MikroORM,
+    private readonly em: EntityManager,
+  ) {}
+
```
Comment (TangoYankee) [inline src/app.service.ts:11]: Because we don't end up using these properties, we can probably remove these changes.

**Review by TangoYankee — CHANGES_REQUESTED** [review]

File: src/app.service.ts (hunk 1 above)
Comment (TangoYankee) [reply src/app.service.ts:11]: It might makes sense to put this branch on the back-burner and focus on getting the endpoints written in drizzle first

Comment (TylerMatteo) [reply src/app.service.ts]: @TangoYankee I'm supportive of that. If you still want to do that, can you convert this PR to draft?

**Comment by TangoYankee** [comment]
Closing as part of #127

**Pull request closed without merging**